README.md

```diff
@@ -1,4 +1,76 @@
+is an ecological observatory, solely funded by the National Science Foundation 
+that will collect and provide open data for 30 years. 
+
+[www.neondataskills.org](http://www.neondataskills.org)
+
+Education is a key component of the NEON mission. As such this site serves to 
+facilitate use of data in both science and education.
+
+---
+
+### Work with Spatio-temporal Data in R -- Data Theme: Phenology
+The original materials were developed during a lesson building hack-a-thon 
+jointly organized by Leah Wasser (NEON), Tracy Teal (Data Carpentry), Mike 
+Smorul (National Socio-Environmental Synthesis Center (SESYNC) and Jason William
+(iPlant Collaborative). 
+
+Hack-a-thon participants included Mike Alonzo, Sean Barberie, Ben Best, 
+Zack Bryn, Jonah Duckles, Marisa Guarinello, Jeff Hollister, Megan A. Jones, 
+Matthew Kwit, Kristina Riemer, Dave Roberts, Keely Roth, Mike Smorul, 
+Courtney Soderberg, Joseph Stachelek, Kaitlin Stack Whitney, Tracy Teal, 
+Leah A. Wasser, Jason Williams, and Meg Williams.
+
+The results of this collaboration are four tutorial 
+series hosted on the NEON Data Skills portal (neondataskills.org) and also 
+taught as Data Carpentry workshops (www.datacarpentry.org).  
+
+* Introduction to Working With Spatio-Temporal Data and Data Management
+* Introduction to Working With Raster Data in R
+* Working With Raster Time Series Data in R
+* Introduction to Working With Time Series Data in Text Formats in R
+
+## Contributors
+Contributors to this repo include Megan Jones, Sarah Newman, Joseph Stachelek, 
+and Leah Wasser.
+
+## Teaching Data Subsets
+All teaching data subsets can be found on the 
+<a href="https://figshare.com/authors/NEON_Data_Skills_Teaching_Data_Subsets/834136" target="_blank"> NEON Data Skills Teaching Data Subsets figshare page </a>. 
+
+### Spatio-temporal Series Teaching Data Subsets
+This data was curated into a teaching data subset by Leah A. Wasser. Original
+sources of the data include
+<a href="http://www.neoninc.org/data-resources/get-data/airborne-data" target="_blank"> NEON Airborne Data</a>,
+the <a href="http://landsat.usgs.gov" target="_blank" > U.S. Geological Survey (USGS) Landsat Satellite</a>, 
+the <a href="http://harvardforest.fas.harvard.edu/" target="_blank">Harvard Forest</a>,
+and the <a href="https://www.census.gov/geo/maps-data/data/tiger-cart-boundary.html" target="_blank">US Census Bureau</a> 
+
+---
+
+## Questions?
+Having a problem getting something to work or want to know why we set up 
+something in a certain way? Email us at neondataskills @ neoninc.org, 
+ping Leah Wasser on Twitter 
+[@leahawasser](http://twitter.com/leahawasser) 
+or 
+[file a GitHub Issue](https://github.com/NEONInclwasser/NEON-Data-Skills/issues).
+And if you have a cool resource that you think would add to the site, please let
+us know!
+
+## License
+The main theme of this site was created by Minimal Mistakes. The theme is free 
```

Comment:
This site uses the Minimal Mistakes these for Jekyll. The theme is open source software ...